distribution/openrct2.d.ts

```diff
@@ -1386,6 +1387,10 @@ declare global {
         readonly crashIntoType: VehicleCrashIntoType;
     }
 
+    interface CalculateGuestCapArgs {
+        suggestedMaxGuests: number;
```

Comment:
Also, nitpick, but to have it named the same as `park.suggestedGuestMaximum`:
```suggestion
        suggestedGuestMaximum: number;
```